Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -9,12 +9,10 @@
 import OpenAI
 import SwiftUI
 
-
```

Comment:
Please re-add the two empty lines between the import statements and the main type.